talk/talk.qmd

```diff
@@ -268,6 +268,31 @@ Ciric et al., 2022, *Nature Methods*
 
 ## Middleware: SDCFlows
 
+- Susceptibility distortion correction is a significant source of analytical flexibility
+    - Multiple fieldmap sequences (spin echo, gradient echo, scanner-reconstructed $B_0$ maps)
+    - Multiple correction implementations with different data structures
+- Comparability among methods is poor, with little insight into sources of variation
+
+- SDCFlows
+    - Defines an intermediate representation of a field inhomogeneity map
+    - Wraps or implements methods for estimating this map from different sequences
+    - Provides a single method for applying estimated maps to EPI sequences
+
+- SDCFlows is used in fMRIPrep and dMRIPrep, with plans to make a standalone tool
+  for custom processing or easier interoperation with other pipelines
+
+
+## Infrastructure: Architectural redesign
+
+- Problem: fMRIPrep's "one size fits all" design has limitations for emerging use cases
```

Comment:
I usually run away from calling this a problem. I believe this is just bringing the future to today (I think I told Russ this "vision" one):

you can see storage, like OpenNeuro, as a place for the original data, which MUST be accessed through filters (except the dataset owners who can access the original data).

Then users may access data through "filters" they are allowed to use. E.g., "defacing & anonymizing" would give you defaced anatomical and remove identifying info. This way, fMRIPrep could become such a filter, and "build the preprocessed dataset" on the spot.